aidial_adapter_bedrock/llm/model/claude3/converters.py

```diff
@@ -0,0 +1,137 @@
+import mimetypes
+from typing import Iterable, List, Literal, Optional, Tuple, Union
+from urllib.parse import urlparse
+
+from aidial_sdk.chat_completion import Attachment, FinishReason, Message, Role
+from anthropic.types import ImageBlockParam, MessageParam, TextBlockParam
+from anthropic.types.image_block_param import Source
+
+from aidial_adapter_bedrock.dial_api.storage import (
+    FileStorage,
+    download_file_as_base64,
+)
+
+ImageMediaType = Literal["image/png", "image/jpeg", "image/gif", "image/webp"]
+IMAGE_MEDIA_TYPES: Iterable[ImageMediaType] = {
+    "image/png",
+    "image/jpeg",
+    "image/gif",
+    "image/webp",
+}
+
+
+def _validate_media_type(media_type: str) -> ImageMediaType:
+    if media_type not in IMAGE_MEDIA_TYPES:
+        raise ValueError(f"Unsupported media type: {media_type}")
+    return media_type  # type: ignore
+
+
+def _create_image_block(
+    media_type: ImageMediaType, data: str
+) -> ImageBlockParam:
+    return ImageBlockParam(
+        source=Source(
+            data=data,
+            media_type=media_type,
+            type="base64",
+        ),
+        type="image",
+    )
+
+
+async def _download_data(url: str, file_storage: Optional[FileStorage]) -> str:
+    if urlparse(url).scheme:
```

Comment:
`url` pointing to a DIAL storage file isn't necessarily a relative URL. 

It might be an absolute URL like `DIAL_URL/v1/files/...`

Reply:
Fixed.